README.md

```diff
@@ -2,6 +2,18 @@
 
 whitehall is a Ruby on Rails content management application for content published by government departments and agencies.
 
+## Running the Application
+
+Startup [using govuk-docker](https://github.com/alphagov/govuk-docker). Once running, `Whitehall` does not have an index, some suggested starting pages are below:
+
+- [`http://whitehall-admin.dev.gov.uk/government/admin/news/new`](http://whitehall-admin.dev.gov.uk/government/admin/news/new)
```

Comment:
I wonder if it might be worth differentiating these 2 links between "publishing view" and "site view" or something similar.

Alternatively, you don't need the square brackets followed by the standard brackets for URLs. You can instead do:

```
<http://whitehall-admin.dev.gov.uk/government/admin/news/new>
```

That's unless you still want the back ticks for code view.

Reply:
Good points 🙌🏼  
I'll update to include all these - 🤔  debating if displaying link should formatting _as_ code or even if I should include this `http:` element